tests/test_examples.py

```diff
@@ -355,6 +355,10 @@ def _sort_by_extension(path):
             except shutil.Error:
                 # Some tests reuse the examples for different checks; ignore errors
                 pass
+        else:
+            # Remove the installer and the install dir at the end of the test
+            shutil.rmtree(str(installer), ignore_errors=True)
+            shutil.rmtree(str(install_dir), ignore_errors=True)
```

Comment:
I think removing `install_dir` is problematic on Windows since Windows installers also add registry keys. It is better to run the uninstaller here. I would say this is out of scope for this PR though. With your name changes, removing `install_dir` this is not necessary anymore.